app/src/main/java/com/mapbox/services/android/navigation/testapp/example/ui/ExampleActivity.kt

```diff
@@ -225,6 +229,10 @@ class ExampleActivity : AppCompatActivity(), ExampleView {
     startActivity(Intent(this, NavigationSettingsActivity::class.java))
   }
 
+  override fun showAttributionDialog(attributionView: View) {
+    AttributionDialogManager(attributionView.context, mapboxMap!!).onClick(attributionView)
```

Comment:
`NavigationMapboxMap` has a `NavigationMapboxMap#retrieveMap` that will return the `MapboxMap` it's wrapping.   So we can remove the need for another variable here, as well as the `mapboxMap!!` if we do something like:
```
map?.retrieveMap()?.let {
    // Attribution code here...
    // in this block "it" will be the MapboxMap
}
```